packages/web/components/table/assets-table-v1.tsx

```diff
@@ -342,6 +343,9 @@ export const AssetsTableV1: FunctionComponent<Props> = observer(
       const data: TableCell[] = [];
       const favorites: TableCell[] = [];
       filteredSortedCells.forEach((coin) => {
+        coin.amount = <PrivateText text={coin.amount} />;
```

Comment:
I know this whole function isn't yours, but this feels strange to me to be mutating the objects instead of making this more 'functional'. We can look at this in person

Reply:
I fixed this by applying this here:
https://github.com/osmosis-labs/osmosis-frontend/blob/6e4ad5a3008eaee7c2009c8df39454a055ee1771/packages/web/components/table/cells/balance.tsx#L13-L17